Update magnetometer wiki to better explain calibration initiation (#248)
betaflight/betaflight.com: docs/wiki/archive/Magnetometer.md

## Magnetometer Calibration

For accurate heading readings, accurate calibration is essential.

Calibration 'zeroes out' local magnetic fields arising from nearby ferrous objects on the quad, such as nearby circuit board components, and any inherent offsets in the sensor.

The calibration process can be initiated while connected to Configurator, or by using stick commands.

> :::important

Reviewer: Afaik we do not have important keyword. Please choose one:

```
:::note

This is a note. It notes things.

:::

:::caution

This is a caution. It cautions things.

:::

:::danger

This is a danger. It is dangerous.

:::

:::info

This is an info. It informs things.

:::

:::tip

This is a tip. It tips things.

:::
```

If we are connected to the Sensors tab, we should see the X axis being close to maximum, and the other two axes close to zero, when the nose is pointing directly into the field. Conversely, it should show the most negative value when the tail points directly into the field.

The resulting cal value for X can be returned with `get mag_calibration` the CLI; the first value is the X cal value. Record that value. Repeat the process to be sure the X cal value is reliable. Then repeat separately for the Y and Z axes, until you have all three cal values. Then you can type in the cal value for each axis into the CLI in the form `set mag_calibration = X,Y,Z`.

> Typically, the arm swinging while rotating method works very well, and it returns a cal value for all three axes in the one calibration run.

Reviewer: ```suggestion
Normally the method of swinging the arm as it rotates works very well, returning a calibration value for all three axes in one calibration run.
```
